src/middleware/socket.ts

```diff
@@ -142,6 +143,7 @@ const chatMiddleware: Middleware = (store) => (next) => (action) => {
   }
 
   if (resendMessage.match(action) && !('$isSync' in action)) {
+    // @ts-ignore must understand why the action is of type never??
     handleMessageResending(action.payload);
   }
```

Comment:
@IlievaDayana What happens if you check existence of `payload` property in `action` before use and what is actual type of `Action` from Redux toolkit?

Reply:
i've tried adding type check also property check and both also, but it did not work?
Action<T> where T stands for template, maybe i should describe it somehow.